protobuf/lib/src/protobuf/field_info.dart

```diff
@@ -231,26 +231,16 @@ class MapFieldInfo<K, V> extends FieldInfo<PbMap<K, V>?> {
 
   final BuilderInfo mapEntryBuilderInfo;
 
-  MapFieldInfo(
-      String name,
-      int tagNumber,
-      int index,
-      int type,
-      this.keyFieldType,
-      this.valueFieldType,
-      this.mapEntryBuilderInfo,
-      this.valueCreator,
-      {ProtobufEnum? defaultEnumValue,
-      String? protoName})
-      : super(name, tagNumber, index, type,
+  MapFieldInfo(String name, int tagNumber, int index, this.keyFieldType,
+      this.valueFieldType, this.mapEntryBuilderInfo, this.valueCreator,
+      {ProtobufEnum? defaultEnumValue, String? protoName})
+      : super(name, tagNumber, index, PbFieldType._MAP,
```

Comment:
Personally I prefer trailing comma for long argument lists:
```
  MapFieldInfo(
    String name,
    int tagNumber,
    int index,
    this.keyFieldType,
    this.valueFieldType,
    this.mapEntryBuilderInfo,
    this.valueCreator, {
    ProtobufEnum? defaultEnumValue,
    String? protoName,
  });
```
YMMV

Reply:
Huh.. didn't know I get to choose whether to put trailing comma or not when using `dart format`. FTR I also prefer trailing commas in long lists.